### ISSUE TITLE
Provide a tab on each content entity to export the entry

### ISSUE THREAD

**Issue by dawehner** [larowlan/default_content#10]
In order to use default_content you somehow need to export the content.

One way, as described in the readme, is to use some REST request.

An alternative way could be to provide a tab on each content entity:

Would you like to have such a feature inside default_content itself?
![default_content](https://cloud.githubusercontent.com/assets/29678/4089591/f32c886a-2f6e-11e4-8b1d-94cfeba80010.png)

**Comment by Berdir** [comment]
Looks interesting. Wondering if it should be a local task or something else, maybe a drush command?

If we do this, we should then also apply the path changes (change every URL to drupal.org, so that it's not specific to your site), which is something you currently have to do manually.

**Comment by dawehner** [comment]
The output itself is pretty straightforward, so it could be easily integrated into a drush command as well. I kinda see default_content as a site builder tools as well, so I ask myself whether having something like a UI would be useful as well.

```
    $entity = $this->entityManager->getStorage($entity_type)->load($entity_id);
    /** @var \Symfony\Component\Serializer\SerializerInterface $serializer */
    $serializer = $this->serializer;
    $output = $serializer->serialize($entity, 'hal_json', ['json_encode_options' => JSON_PRETTY_PRINT]);
```

Personally I would prefer to use the drush command, of cource :)

**Comment by andypost** [comment]
I think it makes sense to have a UI too, but better to place it somewhere in Development.
Implementation could use config single export alike

**Comment by larowlan** [comment]
Yes, pretty sure I had reference somewhere a todo to add this. Great idea

**Comment by dawehner** [comment]
Yes, there is one in the readme
